placebo/serializer.py

```diff
@@ -53,8 +66,13 @@ def serialize(obj):
         result['second'] = obj.second
         result['microsecond'] = obj.microsecond
         return result
-    if isinstance(obj, StreamingBody):
-        result['body'] = obj.read()
+    # Convert objects to dictionary representation based on type
+    if isinstance(obj, botocore.response.StreamingBody):
+        result['payload'] = obj.read() 
+        # Set the original stream to the buffered representation of itself,
+        # so that it can be re-read downstream.
+        obj._raw_stream = StringIO(result['payload'])
```

Comment:
I actually had an argument here that this technically needs to be BytesIO (from what I can tell, its possible to get a binary payload response though I cannot find one).  Will work on tests.